.gitignore

```diff
@@ -7,9 +7,6 @@
 # Ignore bundler config
 /.bundle
 
```

Comment:
You should add `node_modules` to your `.gitignore` so we don't add a ton of files to the repo